clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -648,6 +648,7 @@ export const CustomSelect = ({
             isDisabled={isDisabled}
             isCustomOption={isCustomOption}
             menuPosition={props.menuPosition}
+            onChange={!isFormikOnChange ? onChange : undefined}
```

Comment:
what's the reason for adding this?

Reply:
Whoops, I think that got left in from an earlier iteration where I was using that component.  Good catch.

Reply:
@jpople are you going to remove this?

Reply:
@jpople are you going to create a followup ticket to add cypress tests?

Reply:
Followup ticket https://ethyca.atlassian.net/browse/PROD-1343